docs/usage/configuration-options.md

```diff
@@ -742,6 +742,21 @@ A similar one could strip leading `v` prefixes:
 
 Set this to `false` if you want to disable release notes fetching.
 
+Renovate can fetch release notes when you're running it on one of these platforms:
+
+- GitHub (.com and Enterprise)
+- GitLab (.com and CE/EE)
+- Bitbucket Cloud
+- Bitbucket Server
+- Azure DevOps
+- Gitea
```

Comment:
I've listed all platforms that Renovate can run on, but we cannot fetch release notes on all platforms.

Can you maintainers please check the code and tell me which platforms we can grab release notes from? 😉

Reply:
only on gitlab and GitHub

Reply:
```suggestion
- GitHub (.com and Enterprise)
- GitLab (.com and CE/EE)
```